### PR TITLE
Update Webhook node docs

### PR DESCRIPTION
**Pull request by lorenanda**
- Fixed formatting issues
- Updated/Added node parameters
- Updated screenshots
- Rephrased tutorial instructions, and tweaked the Set expression
- Made links open in new tab

### CODE REVIEW

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 1]
```diff
@@ -1,65 +1,64 @@
 # Respond to Webhook
 
-The *Respond to Webhook* node can be used in workflows with a [Webhook](/integrations/core-nodes/n8n-nodes-base.webhook/) node. It allows controlling the response to incoming webhooks. In the Webhook node, the **Using 'Respond to Webhook' node** option would need to be selected in the **Response** dropdown for the *Respond to Webhook* node to work.
+The *Respond to Webhook* node allows controlling the response to incoming webhooks.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:3]:
```suggestion
The Respond to Webhook node allows you to control the response to incoming webhooks.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 2]
```diff
@@ -1,65 +1,64 @@
...
+
+This node can be used in workflows with a [Webhook node](/integrations/core-nodes/n8n-nodes-base.webhook/){:target="_blank"}. To do this, in the Webhook node select the option **Using 'Respond to Webhook' node** from the **Response** operation.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:5]:
```suggestion
You can use this node in workflows with a [Webhook node](/integrations/core-nodes/n8n-nodes-base.webhook/){:target="_blank"}. To do this, in the Webhook node select the option **Using 'Respond to Webhook' node** from the **Response** operation.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 3]
```diff
@@ -1,65 +1,64 @@
...
 
 !!! note "Expressions"
-    When using [expressions](/code-examples/expressions/), the *Respond to Webhook* node will only run for the first item in the input data
+    When using [expressions](/code-examples/expressions/){:target="_blank"}, the Respond to Webhook node will only run for the first item in the input data.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:8]:
```suggestion
    When using [expressions](/code-examples/expressions/){:target="_blank"}, the Respond to Webhook node only runs for the first item in the input data.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 4]
```diff
@@ -1,65 +1,64 @@
...
 
-- **First Incoming Item**: Respond with the first incoming item's JSON.
-- **Text**: Respond with a text defined in the **Response Body** field.
-- **JSON**: Respond with a JSON object defined in the **Response Body** field.
-- **Binary**: Respond with a binary file defined in the **Response Data Source** field.
-- **No Data**: No response payload is sent.
+The node has a **Respond With** parameter that supports the following modes:
 
-Available options:
+- **First Incoming Item** — Respond with the first incoming item's JSON.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:19]:
```suggestion
- **First Incoming Item** - Respond with the first incoming item's JSON.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 5]
```diff
@@ -1,65 +1,64 @@
...
+- **Text** — Respond with a text defined in the **Response Body** field.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:20]:
```suggestion
- **Text** - Respond with the text defined in the **Response Body** field.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 6]
```diff
@@ -1,65 +1,64 @@
...
+- **JSON** — Respond with a JSON object defined in the **Response Body** field.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:21]:
```suggestion
- **JSON** - Respond with a JSON object defined in the **Response Body** field.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 7]
```diff
@@ -1,65 +1,64 @@
...
+- **Binary** — Respond with a binary file defined in the **Response Data Source** field.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:22]:
```suggestion
- **Binary** - Respond with a binary file defined in the **Response Data Source** field.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 8]
```diff
@@ -1,65 +1,64 @@
...
+- **No Data** — Don't send a response payload.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:23]:
```suggestion
- **No Data** - Don't send a response payload.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 9]
```diff
@@ -1,65 +1,64 @@
...
 
-- **Response Code**: Set the [response code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) to be used.
-- **Response Headers**: Define response headers to be sent.
+### Optional parameters
 
-## Workflow behaviour
+The node has the following options:
 
-When using the *Respond to Webhook* node, workflows will behave as follows:
+- **Response Code** — Set the [response code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status){:target="_blank" .external} to be used.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:29]:
```suggestion
- **Response Code** - Set the [response code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status){:target="_blank" .external} to be used.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 10]
```diff
@@ -1,65 +1,64 @@
...
+- **Response Headers** — Define the response headers to be sent.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:30]:
```suggestion
- **Response Headers** - Define the response headers to send.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 11]
```diff
@@ -1,65 +1,64 @@
...
+- When the workflow finishes without executing the Respond to Webhook node, a standard message is returned with a 200 status.
+- If the workflow errors before the first Respond to Webhook node gets executed, an error message is returned with a 500 status.
+- If there is no webhook set up, the Respond to Webhook node node is ignored in the execution.
 
-- [Webhook](/integrations/core-nodes/n8n-nodes-base.webhook/)
-- [Respond to Webhook]()
+## Example usage
 
-![The workflow using the Respond to Webhook node](/_images/integrations/core-nodes/respondtowebhook/workflow.png)
+[This workflow](https://n8n.io/workflows/1306){:target="_blank"} serves an HTML page when a GET request is made.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:43]:
```suggestion
[This workflow](https://n8n.io/workflows/1306){:target="_blank"} serves an HTML page in response to a GET request.
```

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 12]
```diff
@@ -1,65 +1,64 @@
...
-7. Open the **Test URL** from the *Webhook* node in a new browser tab. The browser should now show the page added defined in the **Response Body** field of the *Respond to Webhook* node.
+This node defines the response to the request received in the previous step.
 
-![The Respond to Webhook node](/_images/integrations/core-nodes/respondtowebhook/respond_to_webhook_node.png)
+1. From the **Respond With** dropdown list, select **Text**.
+2. In the **Response Body** field, enter the HTML code for the page you want to display (for example, the [Bootstrap Starter template](https://getbootstrap.com/docs/5.1/getting-started/introduction/#starter-template){:target="_blank" .external}).
+3. Select **Add Option** > **Response Headers** > **Add Response Header** to add a header to the response with the following values:
+   - **Name:** `Content-Type`
+   - **Value:** `text/html; charset=UTF-8`
+4. Close the *Respond to Webhook* modal and click on **Execute Workflow**.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:63]:
```suggestion
4. Close the **Respond to Webhook** modal and click on **Execute Workflow**.
```

Comment (StarfallProjects) [reply docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:63]: Should this be two steps? You close the modal, then click execute?

File: docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md [hunk 13]
```diff
@@ -1,65 +1,64 @@
...
+5. Open the **Test URL** from the *Webhook* node in a new browser tab. The browser should now show the page defined in the **Response Body** field of the Respond to Webhook node.
```
Comment (StarfallProjects) [inline docs/integrations/core-nodes/n8n-nodes-base.respondToWebhook.md:64]:
```suggestion
5. Open the **Test URL** from the Webhook node in a new browser tab. The browser should now show the page defined in the **Response Body** field of the Respond to Webhook node.
```

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Hi! Thanks for working on this!

I've suggested a bunch of minor style changes.

A couple of additional edits:

- Please replace 'click' with 'select'. The style guide favours select: https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui (and this makes sense with the rise of touchscreens)
- Please optimise the new images (https://tinypng.com/ is quick and free)

Let me know if you have any questions :-)

**Comment by lorenanda** [comment]
Thanks for the review and feedback! I made the suggested edits, hope the explanations and images are better now, but let me know if I should change anything else.

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
I've just merge main into this branch, and it includes a bunch of additional Vale setup. Can I suggest as an exercise:

1. Get Vale setup locally https://www.notion.so/n8n/Style-tooling-764915624637458f872ef81233e7d4ac (I appreciate Vale is not intuitive - please let me know if you need help)
2. Focusing on just these two docs, try to fix as many issues as possible (I wouldn't expect to fix all of them - sometimes Vale needs to be ignored).
3. And also do a style pass, keeping in mind the style guide basics https://github.com/n8n-io/n8n-docs/wiki/Styles

Perhaps also think about the contents generally. I'm not suggesting a full rewrite at this stage, but maybe we can use these examples as a prompt to start talking about templates for this section.

Let's catch up about this on Wednesday?

**Pull request closed without merging**